2024-11-08 12:50:24,305 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'rentals'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'rentals'
2024-11-08 12:50:24,320 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'rentals'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'rentals'
2024-11-08 14:30:03,867 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airplane_mode'
2024-11-08 14:30:06,010 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airplane_mode'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airplane_mode'
2024-11-08 14:30:06,014 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airplane_mode'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airplane_mode'
2024-11-08 16:40:31,802 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'print_designer'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'print_designer'
2024-11-08 16:40:31,808 ERROR frappe Unable to load translations
Site: lap.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/app.py", line 176, in init_request
    frappe.init(site, sites_path=_sites_path, force=True)
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 285, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1731, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1579, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'print_designer'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 164, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 232, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 152, in _merge_translations
    all_translations.update(get_translations_from_apps(lang))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 180, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/translate.py", line 192, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1544, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1574, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sanku-sodhi/lap-bench/apps/frappe/frappe/__init__.py", line 1515, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'print_designer'
2024-11-12 11:56:18,100 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {'docs': '{"name":"Shop","owner":"Administrator","creation":"2024-11-12 11:55:13.060064","modified":"2024-11-12 11:55:13.060064","modified_by":"Administrator","docstatus":0,"idx":0,"read_only":0,"in_create":0,"naming_rule":"","allow_rename":1,"module":"Airport Shop Management","is_submittable":0,"istable":0,"issingle":0,"is_tree":0,"is_calendar_and_gantt":0,"editable_grid":0,"quick_entry":0,"track_changes":0,"track_seen":0,"track_views":0,"custom":0,"beta":0,"is_virtual":0,"queue_in_background":0,"max_attachments":0,"hide_toolbar":0,"allow_copy":0,"allow_import":0,"allow_events_in_timeline":0,"allow_auto_repeat":0,"make_attachments_public":0,"show_title_field_in_link":0,"translated_doctype":0,"sort_field":"creation","sort_order":"DESC","force_re_route_to_default_view":0,"document_type":"","show_preview_popup":0,"show_name_in_global_search":0,"email_append_to":0,"has_web_view":0,"allow_guest_to_view":0,"index_web_pages_for_search":1,"engine":"InnoDB","doctype":"DocType","actions":[],"fields":[{"name":"5l16a5rvtj","owner":"Administrator","creation":"2024-11-12 11:55:13.060064","modified":"2024-11-12 11:55:13.060064","modified_by":"Administrator","docstatus":0,"idx":1,"fieldtype":"Section Break","fieldname":"section_break_ycjm","precision":"","length":0,"non_negative":0,"hide_days":0,"hide_seconds":0,"reqd":0,"is_virtual":0,"search_index":0,"not_nullable":0,"sort_options":0,"show_dashboard":0,"fetch_if_empty":0,"hidden":0,"show_on_timeline":0,"bold":0,"allow_in_quick_entry":0,"translatable":0,"print_hide":0,"print_hide_if_no_value":0,"report_hide":0,"collapsible":0,"hide_border":0,"in_list_view":0,"in_standard_filter":0,"in_preview":0,"in_filter":0,"in_global_search":0,"read_only":0,"allow_on_submit":0,"ignore_user_permissions":0,"allow_bulk_edit":0,"make_attachment_public":0,"permlevel":0,"ignore_xss_filter":0,"unique":0,"no_copy":0,"set_only_once":0,"remember_last_selected_value":0,"columns":0,"parent":"Shop","parentfield":"fields","parenttype":"DocType","doctype":"DocField"}],"links":[],"states":[],"permissions":[{"name":"nv88grha1u","owner":"Administrator","creation":"2024-11-12 11:55:13.060064","modified":"2024-11-12 11:55:13.060064","modified_by":"Administrator","docstatus":0,"idx":1,"role":"System Manager","if_owner":0,"permlevel":0,"select":0,"read":1,"write":1,"create":1,"delete":1,"submit":0,"cancel":0,"amend":0,"report":1,"export":1,"import":0,"share":1,"print":1,"email":1,"parent":"Shop","parentfield":"permissions","parenttype":"DocType","doctype":"DocPerm"}],"__last_sync_on":"2024-11-12T06:26:17.906Z"}', 'method': 'check_pending_migration', 'cmd': 'run_doc_method'}
2024-11-13 12:11:54,342 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
2024-11-13 12:11:54,430 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
2024-11-13 12:12:18,767 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
2024-11-13 12:12:18,854 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
2024-11-13 12:12:20,028 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
2024-11-13 12:12:20,087 ERROR frappe New Exception collected in error log
Site: lap.com
Form Dict: {}
